src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -93,22 +104,15 @@ const DefaultTranscriptslist = (props: Props) => {
       <div className={styles.header}>
         {isFilterOpen && (
           <TranscriptsFilter
+            filterLabel={filterLabel}
```

Comment:
prop name `filterLabel` -> `label`?

Reply:
It's named as `filterLabel` everywhere else so wanted to be consistent with the naming. Do you still think we should change this to `label`?

Reply:
> everywhere else so wanted to be consistent with the naming

I don't understand what you mean by consistency here. TranscriptsFilter component expects a list of properties from the outside world. Inside the TranscriptsFilter component, it doesn't make much sense in prefixing the word "label' with the word "filter", because there seems to be only one label that this component receives from the outside with the props. In the world that lies outside the TranscriptsFilter component, i.e. in the `DefaultTranscriptsList` component, the name `filterLabel` makes a bit more sense (although I still don't like that we refer to the filter and sort box as just filter). There is no requirement for the name of a property of a chid component to match the name of the variable outside that component. My suggestion was to change `filterLabel={filterLabel}` to `label={filterLabel}`. Now that I'm re-reading my last comment, I realise that I didn't make it clear.

Reply:
> My suggestion was to change filterLabel={filterLabel} to label={filterLabel}.

That sounds better :) I was thinking you were telling to change it everywhere.

Reply:
It is changed now ✅